src/Sql/Sql/ManagedDatabase/Services/AzureSqlManagedDatabaseAdapter.cs

```diff
@@ -261,6 +261,51 @@ public void StopManagedDatabaseLogReplay(AzureSqlManagedDatabaseModel parameters
             }
         }
 
+        public void MoveManagedDatabase(MoveCopyManagedDatabaseModel model)
+        {
+            if (OperationMode.COPY.Equals(model.OperationMode))
```

Comment:
you could've just passed the `model.OperationMode` to the API 😄

Reply:
it is an enum, 0 or 1, and I need to pass string "Move", "Copy". Is there a way in c# to do this?

Reply:
riight, i think something like `nameof(OperationMode.COPY)` or `OperationMode.COPY.ToString()` should work (just adjust the enum casing if api requires so)